Sources/NIO/SocketAddresses.swift

```diff
@@ -263,15 +263,15 @@ public enum SocketAddress: CustomStringConvertible {
         var ipv6Addr = in6_addr()
 
         self = try ipAddress.withCString {
-            if inet_pton(AF_INET, $0, &ipv4Addr) == 1 {
+            if inet_pton(NIOBSDSocket.AddressFamily.inet.rawValue, $0, &ipv4Addr) == 1 {
                 var addr = sockaddr_in()
-                addr.sin_family = sa_family_t(AF_INET)
+                addr.sin_family = sa_family_t(NIOBSDSocket.AddressFamily.inet.rawValue)
                 addr.sin_port = in_port_t(port).bigEndian
                 addr.sin_addr = ipv4Addr
                 return .v4(.init(address: addr, host: ""))
-            } else if inet_pton(AF_INET6, $0, &ipv6Addr) == 1 {
+            } else if inet_pton(NIOBSDSocket.AddressFamily.inet6.rawValue, $0, &ipv6Addr) == 1 {
```

Comment:
Do you want these raw pton conversions here, or do you want to use the helpers?

Reply:
No, these should go through `NIOBSDSocket`.  Thanks for catching that!